### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by humuhimi**
just fix a little typo.😅

### CODE REVIEW

**Comment by dereuromark** [comment]
Why is that a typo? Looks like a possessive apostrophe.
https://www.grammarly.com/blog/possessive-case/

File: en/development/dependency-injection.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ The container can leverage factory functions to create objects if necessary::
         return new AuditLogService(...$args);
     });
 
-Factory functions will receive all of the class' resolved dependencies as
+Factory functions will receive all of the class resolved dependencies as
 arguments.
```
Comment (othercorey) [inline en/development/dependency-injection.rst]:
```suggestion
Factory functions will receive all of the resolved dependencies for the class
as arguments.
```

Comment (othercorey) [reply en/development/dependency-injection.rst]: The english does look correct. If that is hard for others to read, maybe this is better.

**Comment by humuhimi** [comment]
oh sorry I may just misunderstanding...
you can close...
latter I will add japanese translation.

**Comment by othercorey** [comment]
@humuhimi sorry, I was wrong. It should be `class's`, but I changed it so easier to read.

**Pull request merged**